test .md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
Please make sure this file is not commited, anything inside .gitignore is ignored by git when pushing to remote, more details :point_right:  [.gitignore](https://www.atlassian.com/git/tutorials/saving-changes/gitignore)

according to project requirements _**"Make sure that the last file is actually ignored by git and is not present in your GitHub repository."**_

Reply:
Hi, could you please check the code again?